notes/provision.md

```diff
@@ -1,70 +1,63 @@
-# Provision a computer
+## Download and install Raspberry Pi Imager
 
-## Image and boot
+[https://www.raspberrypi.org/software/]
 
-1. get the operating system image
-  * download the image for ubuntu server 20.10
-2. create a usb drive
-  * write the image to the usb or sd
-3. boot the usb
-4. install the system
-5. boot the system
+## Download the Raspberry Pi OS 64bit Lite Image
 
-* install balena etcher (mac, pc)
-* install rpi-image (linux)
+[https://downloads.raspberrypi.org/raspios_lite_arm64/images/raspios_lite_arm64-2021-04-09/2021-03-04-raspios-buster-arm64-lite.zip]
 
-```
-login ubuntu/ubuntu
-
-### navigation shell
-1. man app
-2. paths
-### files / directories
-### creating and editing files
-
-### Networking
-1. setup wifi
-2. ip address
-3. LAN / WAN
-4. mac address
-5. NAT addressing
-
-## Update System
+## SDCard
 
-1. update all system packages
-2. install a new package `ssh-server`
-3. run a service
-4. update and upgrade
-5. patching system
+1. Eject or pull the SDCard out of the Pi and put it into the USB SDCard adapter.
+2. Put the USB adapter into your workstation.
 
-## Create User
-1. create new user
+## Raspberry Pi Imager Settings
 
-## Remote Login
+1. Run the Raspberry Pi Imager and access the advanced options menu by typing `control+shift+X`.
 
-1. username
-2. ip address
-3. ssh
+* Values in all caps like YOURNAME are where you put in your setting.
 
-## System Process
-
-1. man ps
-2. daemon
-3. ps aux | grep ssh
-4. start, stop, enable
-
-## Start and Reboot
-1. `sudo reboot now`
+```
```

Comment:
My only suggestion would be to include some screen grabs from the advanced settings menu. This might not be as 1-1 for others as it is for you.  (especially for kids & adults with ADD)

Reply:
Not huge issue though.